packages/interbit-ui-components/src/scss/interbit/_app-bucket.scss

```diff
@@ -0,0 +1,113 @@
+.ibweb-app-bucket {
+  $transition-delay: 200ms;
+  $arrow-size: 10px;
+
+  .ibweb-app-bucket-container {
+    &.open {
+      .ibweb-app-bucket-popover {
+        // display: block;
+        opacity: 1;
+        visibility: visible;
+        transition: opacity $transition-delay linear, visibility 0s linear;
+      }
+      .ibweb-app-bucket-trigger {
+        &:after {
+          opacity: 1;
+          visibility: visible;
+          transition: opacity $transition-delay linear, visibility 0s linear;
+        }
+      }
+    }
+    float: right;
+    margin: 0 auto;
+    position: relative;
+    text-align: center;
+    .ibweb-app-bucket-trigger {
+      position: relative;
+      display: inline-block;
+      cursor: pointer;
+      z-index: 5;
+      &:after {
+        box-sizing: content-box;
+        opacity: 0;
+        visibility: hidden;
+        transition: opacity $transition-delay linear, visibility 0s linear $transition-delay;
+        position: absolute;
+        content: "";
+        width: 0;
+        height: 0;
+        border-style: solid;
+        border-width: 0 $arrow-size $arrow-size $arrow-size;
+        border-color: transparent transparent #ffffff transparent;
+        left: 50%;
+        top: 100%;
+        transform: translate(-50%, 0);
+      }
+    }
+    .ibweb-app-bucket-popover {
+      z-index: 5;
+      position: absolute;
+      border-radius: 6px;
+      top: calc(100% + #{$arrow-size});
+      left: 50%;
+      transform: translate(-50%, 0);
+      opacity: 0;
+      visibility: hidden;
+      transition: opacity $transition-delay linear, visibility 0s linear $transition-delay;
+      .ibweb-app-bucket-item-row {
+        display: block;
+        float: left;
+      }
+      box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
+      background-color: #fff;
```

Comment:
Please replace hard-coded colours with SASS variables. You can add any missing ones you need in `interbit-ui-components/src/scss/interbit/_variables.scss`